app/server/models/User.js

```diff
@@ -174,6 +174,14 @@ var schema = new mongoose.Schema({
     default: false,
   },
 
+  role: {
+    type: String,
+    required: true,
+    enum: {
+      values: ['organizer', 'volunteer', 'participant', 'mentor', 'sponsor', 'investor']
```

Comment:
This should work with "enum: ['organizer', 'volunteer', 'participant', 'mentor', 'sponsor', 'investor']" aswell but we should follow the exemple on line 64.

Reply:
ok, I didn't know that. And yes, I agree we should be consistent and have the same style everywhere :+1: